lib/event/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "vector-event"
+version = "0.1.0"
+authors = ["Vector Contributors <vector@timber.io>"]
+edition = "2018"
+
+[features]
+lua = ["rlua"]
+
+[build-dependencies]
+prost-build = "0.6.1"
+
+[dependencies]
+bytes05 = { package = "bytes", version = "0.5" }
+bytes = { version = "0.4.10" }
+chrono = { version = "0.4.6" }
+derive_is_enum_variant = "0.1.1"
+getset = "0.1.0"
+lazy_static = "1.3.0"
+once_cell = "1.3"
+prost = "0.6.1"
+prost-types = "0.6.1"
+regex = "1.3.9"
+serde_json = { version = "1.0.33" }
+serde = { version = "1.0.80" }
+string_cache = "0.7.3"
+toml = "0.4"
+tracing = "0.1.15"
+rlua = { git = "https://github.com/kyren/rlua", optional = true }
```

Comment:
Missing a pin revision

Reply:
That was copied from the top-level `Cargo.toml`. What commit should I target?